spec/ruby-spec.coffee

```diff
@@ -45,6 +45,38 @@ describe "Ruby grammar", ->
     expect(tokens[4]).toEqual value: ' ', scopes: ['source.ruby']
     expect(tokens[5]).toEqual value: '1', scopes: ['source.ruby', 'constant.numeric.ruby']
 
+  it "tokenizes :: separators", ->
+    {tokens} = grammar.tokenizeLine('File::read "test"')
+    expect(tokens[0]).toEqual value: 'File', scopes: ['source.ruby', 'support.class.ruby']
+    expect(tokens[1]).toEqual value: '::', scopes: ['source.ruby', 'punctuation.separator.method.ruby']
+    expect(tokens[2]).toEqual value: 'read ', scopes: ['source.ruby']
+
+    {tokens} = grammar.tokenizeLine('File:: read "test"')
+    expect(tokens[0]).toEqual value: 'File', scopes: ['source.ruby', 'variable.other.constant.ruby']
+    expect(tokens[1]).toEqual value: ':: ', scopes: ['source.ruby', 'punctuation.separator.method.ruby']
+    expect(tokens[2]).toEqual value: 'read ', scopes: ['source.ruby']
+
+    {tokens} = grammar.tokenizeLine('RbConfig::CONFIG')
+    expect(tokens[0]).toEqual value: 'RbConfig', scopes: ['source.ruby', 'support.class.ruby']
+    expect(tokens[1]).toEqual value: '::', scopes: ['source.ruby', 'punctuation.separator.namespace.ruby']
+    expect(tokens[2]).toEqual value: 'CONFIG', scopes: ['source.ruby', 'variable.other.constant.ruby']
+
+    {tokens} = grammar.tokenizeLine('RbConfig:: CONFIG')
+    expect(tokens[0]).toEqual value: 'RbConfig', scopes: ['source.ruby', 'variable.other.constant.ruby']
+    expect(tokens[1]).toEqual value: ':: ', scopes: ['source.ruby', 'punctuation.separator.namespace.ruby']
```

Comment:
I'd prefer to not include the whitespace in punctuation scope, would you mind adjusting the patterns to not do that?